sync/snap_server.go

```diff
@@ -130,14 +90,23 @@ func (b *snapServer) GetClassRange(ctx context.Context, request *spec.ClassRange
 			return
 		}
 
-		// TODO: Verify class trie
 		ctrie, classCloser, err := s.ClassTrie()
 		if err != nil {
 			yield(nil, err)
 			return
 		}
 		defer classCloser()
 
+		actualRoot, err := ctrie.Root()
+		if err != nil {
+			yield(nil, err)
+			return
+		}
+		if !actualRoot.Equal(classRoot) {
```

Comment:
sorry, I believe, the verification is againts the request. No need to verify internal state.